backend/django_core/static_src/components/page/upload/Login.svelte

```diff
@@ -1,7 +1,7 @@
 <script lang="ts">
     import Chevron from "$icons/Chevron/Index.svelte";
     import Info from "$icons/Info/Index.svelte";
-    import FishGradient from "../../../public/images/fish/fish-gradient.png";
+    import FishGradient from "../../../public/images/fish/fish-gradient.png?url";
```

Comment:
What's this thing? `?url`
this even works?

Reply:
Otherwise vite throws error